Packs/AWS-IAMIdentityCenter/Integrations/AWSIAMIdentityCenter/AWSIAMIdentityCenter.py

```diff
@@ -2,244 +2,427 @@
-                                                                                  'groupName')))
-
-
-def remove_user_from_groups(args, client):  # pragma: no cover
-    membershipID = list_groups_for_user(args, client)
-    response = client.delete_group_membership(
-        IdentityStoreId=f'{IDENTITYSTOREID}',
-        MembershipId=f'{membershipID}'
+        hr_data = f'The membership id {membershipId} has been successfully created.'
+        result = CommandResults(
+            readable_output=hr_data
+        )
+        return_results(result)
+
+
+def get_group_memberships_for_member(args, client, IdentityStoreId):
+    membershipsOfMember = []
+    userID = get_userId_by_username(args, client, IdentityStoreId)['UserId']
+    kwargs = {
+        'IdentityStoreId': f'{IdentityStoreId}',
+        'MemberId': {
+            'UserId': f'{userID}'
+        }
+    }
+    kwargs = remove_empty_elements(kwargs)
+    groups_response = client.list_group_memberships_for_member(**kwargs)
+    user_groups = groups_response.get('GroupMemberships')
+    for group in user_groups:
+        membershipsOfMember.append(group['MembershipId'])
+    return membershipsOfMember
+
+
+def delete_group_membership(args, client, IdentityStoreId):
+    membershipsToDelete = []
+    if args.get('membershipId') and args.get('userName'):
+        return_error("Please provide one of userName and membershipId.")
+    elif args.get('membershipId'):
+        membershipsToDelete = argToList(args.get('membershipId'))
+    elif args.get('userName'):
+        membershipsToDelete = get_group_memberships_for_member(args, client, IdentityStoreId)
+        if membershipsToDelete == []:
+            demisto.results('User is not member of any group.')
+    else:
+        return_error("userName or membershipId must be provided.")
+    if membershipsToDelete != []:
+        for member in membershipsToDelete:
+            response = client.delete_group_membership(
+                IdentityStoreId=f'{IdentityStoreId}',
+                MembershipId=f'{member}'
+            )
+        if response['ResponseMetadata']['HTTPStatusCode'] == 200:
+            hr_data = f'The membership with ids {membershipsToDelete} have been deleted.'
+            result = CommandResults(
+                readable_output=hr_data
+            )
+            return_results(result)
+
+
+def list_group_memberships(args, client, IdentityStoreId):
+    hr_data = []
+    context_data = {}
+    groupId = get_groupId_by_displayName(args, client, IdentityStoreId).get('GroupId')
+    kwargs = {
+        'IdentityStoreId': IdentityStoreId,
+        'GroupId': groupId,
+        'MaxResults': arg_to_number(args.get('limit') or params.get('limit')),
+        'NextToken': args.get('nextToken')
+    }
+    kwargs = remove_empty_elements(kwargs)
+    response = client.list_group_memberships(**kwargs)
+    memberships = []
+    for membership in response.get('GroupMemberships'):
```

Comment:
add a new line after for cluse
```suggestion
    for membership in response.get('GroupMemberships', []):
```